lib/screens/explore_screens/vouch/vouched_tab/vouched_tab.dart

```diff
@@ -29,7 +30,11 @@ class VouchedTab extends StatelessWidget {
                     padding: const EdgeInsets.all(16.0),
                     child: FlatButtonLong(
                       title: 'Vouch for a member',
-                      onPressed: () => {},
+                      onPressed: () async {
+                        await NavigationService.of(context).navigateTo(Routes.vouchForAMember);
```

Comment:
I think we should return a value back from the vouchForAMember screen, letting this page know if it should reload the list or not.

Reply:
@RaulUrtecho  what do u think

Reply:
If there is an information, it must be updated due to an operation that was made on that screen, then yes.
I see there's a event fired after await for the dialog, that is wrong you need fire that event only if the navigation has a result (not null result).

Reply:
Thanks for the review, now it only loads when a value es returned.